experimental/ExteriorAlgebra/ExteriorAlgebra.jl

```diff
@@ -29,12 +34,15 @@ for its second argument, and this function becomes redundant.
 function var_names_for_singular(xs::Union{AbstractVector{<:AbstractString},
                                           AbstractVector{Symbol},
                                           AbstractVector{Char}})
+##??? return string.(xs);  # shorter but equivalent ???
    NumVars = length(xs);
 # This commented out code appears to be a bit slower.
 ##   L = [""  for _ in 1:NumVars];
 ##   for i = 1:NumVars   L[i] = string(xs[i]);  end;
    L = String[];
-   for var in xs   push!(L, string(var));  end;
+    for var in xs
+        push!(L, string(var));
+    end;
```

Comment:
I understand that you are used to using semicolons in your code. But this is highly non-idiomatic for Julia, and I think it would be best to start adapting to the realities of this language and break with this habit here.

Reply:
I can remove them.  I found them a useful visual clue.